test/integration/operator/extension/extension/extension_test.go

```diff
@@ -426,7 +426,7 @@ var _ = Describe("Extension controller tests", func() {
 
 		Eventually(func() error {
 			return testClient.Get(ctx, client.ObjectKeyFromObject(managedResourceRegistrationFoo), &resourcesv1alpha1.ManagedResource{})
-		}).Should(BeNotFoundError())
+		}).WithPolling(1 * time.Second).WithTimeout(5 * time.Second).Should(BeNotFoundError())
```

Comment:
This actually doesn't follow our testing practices, see https://github.com/gardener/gardener/blob/master/docs/development/testing.md#writing-integration-tests.

> Don't set custom (high) timeouts and intervals in test code: https://github.com/gardener/gardener/pull/4983
iInstead, shorten sync period of controllers, overwrite intervals of the tested code, or use fake clocks: [example test](https://github.com/gardener/gardener/blob/7c4031a57836de20758f32e1015c8a0f6c754d0f/test/integration/resourcemanager/managedresource/resource_suite_test.go#L137-L139)

Reply:
Thanks for the links. I saw the timeouts in the IDE, as I have not set the `GOMEGA_DEFAULT_EVENTUALLY_*` env variables. The defaults values for the integration tests are sufficient.